### PR TITLE
Update game over message

### PR DESCRIPTION
**Pull request by emi-gardav**
Update the game over message so people know how to play again!

### CODE REVIEW

File: README.md
Comment (emi-gardav) [inline README.md:1]: This is great for the user!

**Review by emi-gardav — COMMENTED** [review]
This is great for the user!

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (EmilianoG6) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Nice

Comment (emi-gardav) [inline index.html]:
```suggestion
<h2 hidden>GAME OVER, refresh to play again 🧑‍💻 🤖!</h2>
```

**Pull request merged**